VK_V1.0/VK_V1.0/SessionInformation.swift

```diff
@@ -0,0 +1,16 @@
+// SessionInformation.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Информация о текущей сессии пользователя
+final class SessionInformation {
+
+    // MARK: - Public property
+    static let instance = SessionInformation()
```

Comment:
```suggestion
    static let shared = SessionInformation()
```
в паттерне принято объект синглтона называть shared